app/src/components/items/uv/universalViewer.tsx

```diff
@@ -17,13 +17,15 @@ export type UniversalViewerProps = {
 // https://codesandbox.io/p/sandbox/uv-nextjs-example-239ff5?file=%2Fcomponents%2FUniversalViewer.tsx%3A39%2C1-49%2C8
 const UniversalViewer: React.FC<UniversalViewerProps> = React.memo(
   ({ manifestId, canvasIndex, onChangeCanvas, config }) => {
+    // console.log("canvasIndex is: ", canvasIndex)
+
     console.log("canvasIndex in UniversalViewer component is: ", canvasIndex);
     const ref = useRef<HTMLDivElement>(null);
     const lastIndex = useRef<number>();
     const options = useMemo(
       () => ({
         manifest: manifestId,
-        canvasIndex: canvasIndex || 0,
+        canvasIndex: 2, //canvasIndex || 0,
```

Comment:
Is this on purpose? If so, could you add a comment why?

Reply:
nope this was to demo behavior and I accidentally left it in